programs/sample-host/sample-host.c

```diff
@@ -58,17 +58,17 @@ static void on_client_connect(ziti_connection clt, int status) {
     }
 }
 
-static void on_client(ziti_connection serv, ziti_connection client, int status) {
+static void on_client(ziti_connection serv, ziti_connection client, int status, ziti_client_ctx *clt_ctx) {
     if (status == ZITI_OK) {
         const char *source_identity = ziti_conn_source_identity(client);
```

Comment:
should this sample show using ->caller_id?

Reply:
done